<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>256</width>
     <height>581</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>PathTree</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>9</y>
     <width>521</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>569</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Dir:</string>
   </property>
  </widget>
  <widget class="QLabel" name="curDir">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>570</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>dir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
